content/en/docs/tasks/debug/debug-cluster/_index.md

```diff
@@ -314,6 +314,14 @@ This is an incomplete list of things that could go wrong, and how to adjust your
   - Mitigates: Node shutdown
   - Mitigates: Kubelet software fault
 
+ {{<note>}} 
+Starting 1.24, since management of the CNI is no longer in scope for kubelet, the `--network-plugin` command line parameter has been removed from the binary and will throw the following error, if used:
```

Comment:
```suggestion
From Kubernetes 1.24,the kubelet always uses CNI mode for networking. The legacy `--network-plugin`
command line parameter has been removed from the binary. If you try to specify `--network-plugin`, you see:
```

Reply:
> Maybe we should also update https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/troubleshooting-cni-plugin-related-errors/ to mention that the kubelet no longer needs a `--network-plugin` command line argument.

Would mentioning the exact line works fine or should it be modified for [mentioning here](https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/troubleshooting-cni-plugin-related-errors/)?

Reply:
You are welcome to decide what approach you prefer.